docs/documentation/quartz-3.x/how-tos/reschedule-jobs.md

```diff
@@ -0,0 +1,49 @@
+# Reschedule Jobs
+
+A few ways to approach a need to reschudel a job.
+
+## Manually Retry
+
+When a Quartz job is running, and an unhandled exception escapes the `IJob`, the Quartz system will mark the job in an error state. This would then allow you to reschedule the job using any method that would be work for your system. 
+
+## Polly Retries
+
+If your job simply needs to retry its work, then you could wrap the job in a [Polly](https://github.com/App-vNext/Polly) policy, and use the policy definitions to retry it.
```

Comment:
Maybe add a note about long retry loops inside jobs can be potentially harmful as they can exhaust resources if there's thread pool starvation.